File: programmers/pg_lv1_삼총사.js
function solution(number) {
  let count = 0;

  for (i = 0; i < number.length - 2; i++) {
    for (j = i + 1; j < number.length - 1; j++) {
      let twoSum = number[i] + number[j];
      for (k = j + 1; k < number.length; k++) {
        if (twoSum == -number[k]) count++;
      }
    }
  }

  return count;
}

File: python/programmers/pg_lv1_삼총사.py
def solution(number):
    n = len(number)
    result = 0
    
    for i in range(n - 2):
        sum = number[i]
        for j in range(i + 1, n - 1):
            sum += number[j]
            for k in range(j + 1, n):
                if sum + number[k] == 0: result += 1
            sum -= number[j]
            
    return result